website/blog/testing.md

```diff
@@ -0,0 +1,512 @@
+# Lower Test Toil With Better Local Testing
+
+TODO(mwhittaker): Write an intro paragraph. Describe, briefly, the three kinds
```

Comment:
TODO: Explain that all three kinds of tests are useful.

Reply:
This is a great idea. 

Also, at the end of this maybe we can mention various types of testing packages we provide in weaver for that. And say that we will elaborate on each of them in the rest of the blog post